src/DicomMetaDictionary.js

```diff
@@ -143,9 +143,14 @@ class DicomMetaDictionary {
 
                 if (naturalDataset[naturalName].length === 1) {
```

Comment:
@wayfarer3130 
Apart from circular reference there is a critical change to consider.
From 0.18.8 to 0.18.9 (minor release), has occurred a breaking change. I.e 
Data type for  single sequences is different to  non single sequences.

@pieper  is this expected from lib point of view? Is there a way to avoid having breaking changes released under minor releases?

Reply:
> @pieper is this expected from lib point of view? Is there a way to avoid having breaking changes released under minor releases?

I hadn't realized this was a breaking change - we should reconsider this.  I'm leaning toward removing the concept of any dicom `*Sequence` being anything but an array.  It would mean more `[0]`s in the code but it would be more consistent with the standard.  Since this would break the API it should be a new minor release at least.